src/subscription/subscription.controller.ts

```diff
@@ -28,6 +28,7 @@ export class SubscriptionController {
     const subToken = await this.subscriptionService.subscribe(createSubDto);
     return ZaLaResponse.Ok(subToken, 'User now subscribed to this API', '201');
   }
+  
```

Comment:
@alahirajeffrey where is the guard that is supposed to do the work

Reply:
> @alahirajeffrey where is the guard that is supposed to do the work

There's no Request params for that route

Reply:
@PhilemonBrain I would have to refactor the code because the apiId and profileId are passed in a dto instead of as a parameter. 

Should I go ahead and refactor it?